packages/base/src/Identifier/identifier.ts

```diff
@@ -137,9 +137,8 @@ export class ECKeyIdentifier extends Identifier {
         )
     }
     declare [Symbol.toStringTag]: string
-    #ec: undefined
```

Comment:
I think this change should not be landed. I can provide help with toolchain on firefly side.